sessions/css-animations/toast-animations/js/index.js

```diff
@@ -1,28 +1,32 @@
 console.clear();
 
-const submitButton = document.querySelector('[data-js="submit"]');
+const startButton = document.querySelector('[data-js="start-button"]');
 const toast = document.querySelector('[data-js="toast"]');
-let timeoutId = null;
+const countdown = document.querySelector('[data-js="countdown"]');
 
-toast.addEventListener("click", () => {
-  clearTimeout(timeoutId);
-  exitToast();
-});
+// Sets timeoutId initially to undefined.
+let timeoutId;
 
-submitButton.addEventListener("click", () => {
-  enterToast();
-  if (timeoutId === null) {
-    timeoutId = setTimeout(exitToast, 5000);
+startButton.addEventListener("click", () => {
+  // We do not allow setTimeout to be called again
+  // before it has called hideToast.
+  // When hideToast is called, timeoutId is set to undefined
+  if (timeoutId === undefined) {
+    timeoutId = setTimeout(hideToast, 2000);
+    toast.classList.remove("toast--hidden");
+    countdown.classList.add("countdown--visible");
   }
```

Comment:
I personally would keep the initial value of timeoutId being set to "null", since this indicates that the value is set on purpose by a developer and doesn't results from JS not finding a value.

Reply:
We discussed this in the team and agreed that programmers do not quite agree on this. :D